python/tests/compat/test_mantid.py

```diff
@@ -453,6 +453,17 @@ def test_detector_positions(self):
                 np.isclose(moved_det_position.values,
                            unmoved_det_positions.values)))
 
+    def test_to_quat_from_vectors(self):
+        a = np.array([1, 0, 0])
+        b = np.array([0, 1, 0])
+        q_var = mantidcompat._quat_from_vectors(a, b)
+        q = q_var.value
+        self.assertTrue(np.allclose(b, q.to_rotation_matrix().dot(a)))
```

Comment:
What is the tolerance on `allclose`? I think it's fine as long as the numbers are in the range [0-1].